README.md

```diff
@@ -127,7 +127,7 @@ db.execute('PRAGMA journal_mode = MEMORY;'); // or OFF
 
 If you use [prepared statements](#prepared-statements) are useful to reduce the time of critical queries.
 
-# Perf flag
+# Perf flags
 
 You can turn on the performance flag to tweak all possible performance enhancing compilation flags, this greatly affects performance of sqlite itself:
```

Comment:
Can you modify this a bit and add something like "Be aware this disables thread safety, you should only uses transactions (which operate based on a mutex) to avoid any issues.